openpype/hosts/houdini/api/lib.py

```diff
@@ -558,19 +558,33 @@ def get_frame_data(node):
         dict: frame data for star, end and steps.
 
     """
+    if asset_data is None:
+        asset_data = {}
+
     data = {}
 
     if node.parm("trange") is None:
-
+        self.log.debug(
+            "Node has no 'trange' parameter: {}".format(node.path())
+        )
         return data
 
     if node.evalParm("trange") == 0:
-        self.log.debug("trange is 0")
+        self.log.debug(
+            "Node '{}' has 'Render current frame' set. "
+            "Time range data ignored.".format(node.path())
+        )
         return data
 
-    data["frameStart"] = node.evalParm("f1")
-    data["frameEnd"] = node.evalParm("f2")
-    data["steps"] = node.evalParm("f3")
+    data["frameStartHandle"] = node.evalParm("f1")
+    data["frameStart"] = node.evalParm("f1") + asset_data.get("handleStart", 0)
```

Comment:
I am not quite sure if this PR would also sets the timeline in a right way but...
As far as I know, Houdini has the timeline with` animationStart`, `frameStart`, `frameEnd`, `animationEnd`, which are smiliar to the frame range function in Maya. 
![image](https://github.com/ynput/OpenPype/assets/64118225/5974ebfb-fcf9-4cc1-a79a-f0711ec1900d)

You can take reference from this function 
https://github.com/ynput/OpenPype/blob/4bcad094e2c2bad2f973b266d9bf4878dd3fde63/openpype/hosts/maya/api/lib.py#L2302-L2399